### ISSUE TITLE
Hotkeys for conflict resolution

### ISSUE THREAD

**Issue by janpio** [refined-github/refined-github#6387]
### Description

Currently GitHub's conflict resolution feature is quite cumbersome to use, as the buttons/links are quite far apart. Hotkeys would help a lot to speed up the conflict resolution time. 

Example for the UI:
<img width="1197" alt="image" src="https://user-images.githubusercontent.com/183673/221019876-f3023bdf-1886-467a-af86-0b1d83b3f31e.png">

The usual workflow is to click "Next" top right, then at the bottom of the text field have click "Accept incoming change", then "Next" again until all conflicts are resolved and one can click "Mark as resolved". After all files are resolved from conflicts, one can click "Commit merge" to commit the resolved files.

### Example URLs

https://github.com/prisma/prisma/pull/18020/conflicts

**Comment by fregante** [comment]
Agreed! We should match VSCode's shortcuts, if any and not already in use.

**Comment by janpio** [comment]
Ha, I wasn't even aware part of that comes from Refined GitHub already! Neat.
https://github.com/refined-github/refined-github/blob/main/source/resolve-conflicts.ts

**Comment by fregante** [comment]
The feature is now native
